CMakeLists.txt

```diff
@@ -29,20 +29,53 @@ add_library(
     src/nwb/ecephys/ElectricalSeries.cpp
     src/nwb/file/ElectrodeGroup.cpp
     src/nwb/file/ElectrodeTable.cpp
-    src/nwb/hdmf/base/Data.hpp
     src/nwb/hdmf/base/Container.cpp
     src/nwb/hdmf/table/DynamicTable.cpp
-    src/nwb/hdmf/table/ElementIdentifiers.hpp
-    src/nwb/hdmf/table/VectorData.hpp
+)
+
+add_library(aqnwb::aqnwb ALIAS aqnwb_aqnwb)
+
+include(GenerateExportHeader)
```

Comment:
I wasn't aware of the GenerateExportHeader feature. Will be useful to document how a user is supposed to use these.

Reply:
Yes I came across this when building the shared library, but from my understanding this generates macros like AQNWB_EXPORT that we can add to objects that we want to expose if we're building a shared library. 

I can make an issue to document once I get a chance to incorporate and fully test it out?

Reply:
> I can make an issue to document once I get a chance to incorporate and fully test it out?

Dealing with that as a separate issue is fine